tests/kernel/fatal/exception/testcase.yaml

```diff
@@ -22,5 +22,6 @@ tests:
   kernel.common.stack_sentinel:
     extra_args: CONF_FILE=sentinel.conf
     # FIXME: See issue #39948
-    platform_exclude: qemu_cortex_a9
+    # FIXME: Issue to be filed for PR #45570
+    platform_exclude: qemu_cortex_a9 qemu_xtensa
```

Comment:
The related a9/a53 issue is closed and seems to point at a qemu issue. Is that the case here as well for stack sentinel?